bower.json

```diff
@@ -1,38 +1,37 @@
 {
   "name": "paper-pagination",
-  "version": "0.2.2.1",
-  "description": "A material design element that generate pagination bar",
+  "version": "2.0.2",
+  "description": "A material design element inspire by Danny Phung that generate pagination bar",
   "authors": [
-    "Dong Hung Phung"
+    "Benoit Claveau <benoit.claveau@gmail>"
```

Comment:
`[ "Dong Hung Phung" , "Benoit Claveau <benoit.claveau@gmail>" ]` no?

Reply:
Of course. Excuse me.